csrc/ir_nodes.cpp

```diff
@@ -1398,6 +1397,19 @@ MmaOptions::MmaInputLayout getInputLayout(
     const MmaOp::AxesData& m_axes,
     const MmaOp::AxesData& n_axes,
     const MmaOp::AxesData& k_axes) {
+  // ordered by axis of reduction
+
+  // NT layout (b - broadcast, r - reduction):
+  // A = [K, M, b]
+  // B = [K, b, N]
+  // C = [r, M, N]
+  if ((k_axes.front() < in_a.bcasts.front()) &&
+      (m_axes.front() < in_a.bcasts.front()) &&
+      (k_axes.front() < in_b.bcasts.front()) &&
+      (in_b.bcasts.front() < n_axes.front())) {
+    return MmaOptions::MmaInputLayout::NT;
+  }
```

Comment:
Why this is moved?

Reply:
just want to "ordered by axis of reduction". I think we should be consistent regarding the order of these different layouts. Or the original order is based on frequency of these layouts? This is not related to split-k, I can reverse the change.

Reply:
Related doc: https://github.com/NVIDIA/Fuser/issues/203